libs/designsystem/button/src/button.component.spec.ts

```diff
@@ -87,6 +87,46 @@ describe('ButtonComponent', () => {
       });
     });
 
+    describe('when aria-disabled=true', () => {
+      beforeEach(() => {
+        element.ariaDisabled = 'true';
+      });
+
+      it('should render with correct background-color', () => {
+        expect(element).toHaveComputedStyle({
+          'background-color': getColor('semi-light'),
+        });
+      });
+
+      it('should render with correct color', () => {
+        expect(element).toHaveComputedStyle({
+          color: getColor('semi-dark', 'shade'),
+        });
+      });
```

Comment:
These are a little redundant with the visual tests, but I can see how it follows our approach everywhere else. 

So not important, but consider removing these to not keep piling `toHaveComputedStyle` tests on top.